packages/react/test/errorboundary.test.tsx

```diff
@@ -537,47 +537,47 @@ describe('ErrorBoundary', () => {
       expect(mockOnReset).toHaveBeenCalledTimes(1);
       expect(mockOnReset).toHaveBeenCalledWith(expect.any(Error), expect.any(String), expect.any(String));
     });
+    it.only.each`
```

Comment:
This we should most definitely remove :) 

```suggestion
    it.each`
```

Reply:
fixed in 9c1cf8391d5c959a6c77509581ca7f87b3a1ee3c